src/CHANGES.txt

```diff
@@ -185,6 +185,9 @@ RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
     - Typo in customized decider example in user guide
     - Replace usage of unittest.TestSuite with unittest.main() (fix #3113)
 
+  From Arda Fu
+    - Fix cpp scanner regex logic to treat ifndef. Previously it was not properly 
```

Comment:
Can you add that this issue only shows with python 3.5+?

Reply:
You mean like this?

> - Fix cpp scanner regex logic to treat ifndef for python 3.5+. Previously 
      it was not properly differentiating between if, ifdef, and ifndef.